factory-monitor/node_types/custom-node.c

```diff
@@ -301,11 +301,11 @@ custom_pool_new_type(const struct sol_flow_node_type **current)
      */
     static const struct sol_flow_static_conn_spec conns[] = {
         { 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__OUT__SET_DEVICE_ID, 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__IN__DEVICE_ID },
-        { 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__OUT__SET_FAILURE, 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__IN__IN_FAILURE },
+        { 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__OUT__SET_FAILURE, 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__IN__FAILURE },
 
-        { 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__OUT__OUT_FAILURE, 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__IN__IN_FAILURE },
-        { 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__OUT__OUT_TEMPERATURE, 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__IN__IN_TEMPERATURE },
-        { 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__OUT__OUT_NAME, 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__IN__IN_NAME },
+        { 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__OUT__FAILURE, 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__IN__IN_FAILURE },
+        { 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__OUT__TEMPERATURE, 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__IN__IN_TEMPERATURE },
+        { 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__OUT__NAME, 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__IN__IN_NAME },
```

Comment:
May I do that on a different commit? I didn't want to mix build fix with other stuff...